resources/js/visual-editor/blocks/verse/verse.css

```
@@ -0,0 +1,17 @@
+/*
+ * Verse — frontend styles.
+ *
+ * Ported from `@wordpress/block-library/src/verse/style.scss` (v9.43.0).
+ */
+
+pre.wp-block-verse {
+    box-sizing: border-box;
+    overflow: auto;
+    white-space: pre-wrap;
+    min-width: 1em;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)